README.md

```diff
@@ -486,6 +486,9 @@ pairwise constructs as found in e.g. `let` and `cond`.
     (ns examples.ns
       (:use clojure.zip))
     ```
+* <a name="sort-required-namespaces"></a>
+  Sort required namespaces lexicographically.
```

Comment:
I'd add some example here and also the rationale you've outline in the PR itself. Rules without rationale are like acargo cult. :-)